website/default_config.py

```diff
@@ -46,10 +46,10 @@
         }
     },
     'dev': {
-        'tag': '2.1.0.dev3',
-        'version': (2, 1, 0, 'dev', 3),
+        'tag': '2.1.1.dev1',
+        'version': (2, 1, 1, 'dev', 1),
         'urls': {
-            'download': 'https://github.com/metabrainz/picard/releases/tag/release-2.1.0dev3',
+            'download': 'https://github.com/metabrainz/picard/releases/tag/release-2.1.1dev1',
```

Comment:
There is no such release and IMHO it does currently not make sense to tag this. I'd rather set this to the same as the current release and only update it to a Dev release we actually have.

Reply:
https://github.com/metabrainz/picard/releases/tag/release-2.1.1dev1

Reply:
Ok :)